app/ui.R

```diff
@@ -4,16 +4,31 @@ library(leaflet)
 library(dygraphs)
 library(plotly)
 library(renv)
-library(shinycssloaders) # withSpinner() indicate calculation/rendering
 library(shinythemes) 
+library(waiter) #loading screen/spinner
 
 
 options(shiny.sanitize.errors = TRUE)
 linebreaks <- function(n){HTML(strrep(br(), n))}
 
+# Spinner Stuff------------
+# a notorious gif
+gif <- paste0("https://media1.tenor.com/images",
+              "/cb27704982766b4f02691ea975d9a259/tenor.gif?itemid=11365139")
+
+loading_screen <- tagList(
+  h3("Give me a sec......", style = "color:gray;"),
+  img(src = gif, height = "200px")
+)
+#--------------------------
+  
```

Comment:
if this is that gif, it should be removed

Reply:
sigh....k, going back to use the spinning circle instead of gif